### ISSUE TITLE
[Bug]: Plane - new install - sign up page

### ISSUE THREAD

**Issue by jocubeit** [coollabsio/coolify#2870]
### Description

On brand new one-click install of Plane, using the sslip.io url provided one cannot sign up or login:
Error message:
Instance not configured. Please contact your administrator.

What configuration is required beyond one-click install? I notice the URLs are empty in ENV VARS.

### Minimal Reproduction (if possible, example repository)

Add a service.
Choose Plane.
Deploy.
Go to sslip.io url.
Click on Sign up.
Provide email address.
Click Continue button.

### Exception or Error

Instance not configured. Please contact your administrator.

<img width="490" alt="Screenshot 2024-07-18 at 7 53 32 PM" src="https://github.com/user-attachments/assets/da4cfabc-40dc-4215-b7d4-0190683e6c4e">

### Version

v4.0.0-beta.315

**Comment by andrasbacsai** [comment]
I think you need to access it on `/god-mode` to get started.

**Comment by jocubeit** [comment]
Ok great thank you.

Now what do we need to do to configure a domain and SSL. For every other app I have simply created a traefik dynamic proxy config like so:

```yaml
http:
  routers:
    dashboard:
      rule: Host(`sub.domain.com`)
      service: projects
      entryPoints:
        - https
      tls:
        certresolver: letsencrypt
```
And then add the appropriate sub.domain A record pointing at the server IP in DNS settings.

This does not seem to work with the Plane one-click. The SSL cert is generated, but every URL results in a 404 Not Found HTTP error.

I really think the Coolify docs need to be updated for each service, so we can just follow the docs. Currently the docs seems to end at the Minio service.

**Comment by andrasbacsai** [comment]
Do you use the integrated proxy (Traefik / Caddy)? If yes, certificates and proxies generated automatically for you.

**Comment by action-simon** [comment]
I am not sure if i should make a new topic as it kinda fits. Can I somehow configure it that https://plane.tld/ always redirects to https://www.plane.tld? If I try to enter both urls, I am getting a warning that I should not do that. For example on a GitHub imported page I can easily set this up.

**Comment by jocubeit** [comment]
> Do you use the integrated proxy (Traefik / Caddy)? If yes, certificates and proxies generated automatically for you.

Yes I am just using traefik as it comes in Coolify.
I deleted my dynamic traefik config and it's now working. I didn't know the proxies were automatically being created for me, I thought I had to add my own.

Thank you for asking the right questions and pointing me in the right direction. Very much appreciated.

@andrasbacsai I'm happy to close the issue as resolved, but not sure what you want to do about the above query from @action-simon.
